### ISSUE TITLE
ArgumentError while executing the rollback

### ISSUE THREAD

**Issue by matthiasdeblock** [openvstorage/framework#818]
On GIG's environment we had the following error while executing the rollback:

Version: 2.7.1-fargo.2-1

**Error:**

```
In [23]: VDiskController.rollback('87b309b9-3499-4f5f-bd8a-421e6286e7a0','1471438801')
---------------------------------------------------------------------------
ArgumentError                             Traceback (most recent call last)
<ipython-input-23-4d7e3ca8d065> in <module>()
----> 1 VDiskController.rollback('87b309b9-3499-4f5f-bd8a-421e6286e7a0','1471438801')

/usr/lib/python2.7/dist-packages/celery/local.pyc in <lambda>(x, *a, **kw)
    165     __ge__ = lambda x, o: x._get_current_object() >= o
    166     __hash__ = lambda x: hash(x._get_current_object())
--> 167     __call__ = lambda x, *a, **kw: x._get_current_object()(*a, **kw)
    168     __len__ = lambda x: len(x._get_current_object())
    169     __getitem__ = lambda x, i: x._get_current_object()[i]

/usr/lib/python2.7/dist-packages/celery/app/task.pyc in __call__(self, *args, **kwargs)
    418             if self.__self__ is not None:
    419                 return self.run(self.__self__, *args, **kwargs)
--> 420             return self.run(*args, **kwargs)
    421         finally:
    422             self.pop_request()

/opt/OpenvStorage/ovs/lib/vdisk.pyc in rollback(vdisk_guid, timestamp)
    395         snapshotguid = snapshots[0]['guid']
    396         VDiskController._wait_for_snapshot_to_be_synced_to_backend(vdisk_guid=vdisk.guid, snapshot_id=snapshotguid)
--> 397         vdisk.storagedriver_client.rollback_volume(str(vdisk.volume_id), snapshotguid)
    398         vdisk.invalidate_dynamics(['snapshots'])
    399         return True

ArgumentError: Python argument types in
    StorageRouterClient.rollback_volume(StorageRouterClient, str, unicode)
did not match C++ signature:
    rollback_volume(volumedriverfs::PythonClient {lvalue}, std::string volume_id, std::string snapshot_id='')

```

**Fix:**
/opt/OpenvStorage/ovs/lib/vdisk.py +397
- from

```
vdisk.storagedriver_client.rollback_volume(str(vdisk.volume_id), snapshotguid)
```
- to: 

```
vdisk.storagedriver_client.rollback_volume(str(vdisk.volume_id), str(snapshotguid))
```

**Comment by wimpers** [comment]
@matthiasdeblock can you create a PR for this one?

**Comment by matthiasdeblock** [comment]
@wimpers, created PR on this one.

**Comment by khenderick** [comment]
Fixed by #841, packaged in openvstorage-2.7.2-rev.3884.42ef467
